docs/tutorials/BuildMultiClusterApplication/BuildMultiClusterApplication.md

```diff
@@ -175,21 +183,13 @@ nginx-profile   multi-cluster-app-nginx-profile.trafficmanager.net   True
 
 > Note:  nginx-backend file located [here](./testfiles/nginx-backend.yaml)
 
-```yaml
+```bash

```

Comment:
yaml?

Reply:
i used "..." to skip some of the fields, so it's not a valid yaml file. :(